src/main/resources/jenkins/metrics/api/MetricsAccessKey/resource.js

```diff
@@ -62,3 +62,9 @@ function saveApiToken(button) {
     }
   });
 }
+
+Behaviour.specify(".metrics-generate-api-token", "MetricsAccessKey_generateToken", 0, (element) => {
+  element.addEventListener("click", (event) => {
+    saveApiToken(event.target);
```

Comment:
Ideally this function is either renamed or hidden inside IIFE so that it doesn't exist in global scope on the Global Config page. Name is a little too generic.